src/enzo/Grid_ZeusSolver.C

```diff
@@ -192,6 +215,45 @@ int grid::ZeusSolver(float *gamma, int igamfield, int nhy,
   }
 		 
   /*   1) Add source terms */
+  /* FDM: if FDM is used */
+  if (QuantumPressure){
+    // get code units
+  float TemperatureUnits = 1, DensityUnits = 1, LengthUnits = 1,
+    VelocityUnits = 1, TimeUnits = 1;
+
+  if (QuantumGetUnits(&DensityUnits, &LengthUnits, &TemperatureUnits,
+         &TimeUnits, &VelocityUnits, Time) == FAIL) {
+    ENZO_FAIL("Error in GetUnits.");
+  }
+
+  FLOAT a = 1, dadt;
+  if (ComovingCoordinates)
+      if (CosmologyComputeExpansionFactor(Time+0.5*dtFixed, &a, &dadt) 
+    == FAIL) {
+  ENZO_FAIL("Error in CosmologyComputeExpansionFactors.");
+      }
+  // calculate hbar/m
+  float hmcoef = 5.9157166856e27*TimeUnits/pow(LengthUnits,2)/FDMMass;
```

Comment:
as w/previous comment, should this be double and POW()?

Reply:
doubled and POW()ed